### PR TITLE
shared/install: several cleanups

### PR DESCRIPTION
**Pull request by YHNdnzj**
Split out from #31588

### CODE REVIEW

File: src/shared/install.c [hunk 1]
```diff
@@ -332,122 +333,166 @@ void install_changes_free(InstallChange *changes, size_t n_changes) {
...
+        case INSTALL_CHANGE_DESTINATION_NOT_PRESENT:
+                return log_warning("Unit %s is added as a dependency to a non-existent unit %s.",
+                                   change->source, change->path);
+
+        case INSTALL_CHANGE_AUXILIARY_FAILED:
+                return log_warning("Failed to enable auxiliary unit %s, ignoring.", change->path);
+
+        default:
+                assert_not_reached();
+
```
Comment (poettering) [inline src/shared/install.c]: spurious empty line

File: src/shared/install.c [hunk 2]
```diff
@@ -332,122 +333,166 @@ void install_changes_free(InstallChange *changes, size_t n_changes) {
...
+                break;
+
+        case -EUNATCH:
+                m = strjoin("Cannot resolve specifiers in unit ", change->path);
+                bus_error = BUS_ERROR_BAD_UNIT_SETTING;
+                break;
+
+        default:
+                return change->type;
+
```
Comment (poettering) [inline src/shared/install.c]: ditto

File: src/shared/install.c [hunk 3]
```diff
@@ -332,122 +333,166 @@ void install_changes_free(InstallChange *changes, size_t n_changes) {
...
+        }
+
```
Comment (poettering) [inline src/shared/install.c]: i'd remove this empty line too. i mean, we have this rule that function calls and their immediate error handling should stay together, hence i'd say that rule here means that this empty line should go

**Comment by poettering** [comment]
tried really hard to find something i didn't like. only found empty lines ;-)

ci is a bit unhappy? probably unrelated?

lgtm

File: src/shared/install.c (hunk 1 above)
Comment (YHNdnzj) [reply src/shared/install.c]: Hmm, I have always been writing `switch`s like this. I thought it pairs well with the leading empty line 🤔

Comment (YHNdnzj) [reply src/shared/install.c]: (still updated with suggested change though)

**Pull request merged**